modules/apps/asset/asset-browser-web/src/main/resources/META-INF/resources/toolbar.jsp

```diff
@@ -47,7 +47,7 @@ AssetRendererFactory<?> assetRendererFactory = AssetRendererFactoryRegistryUtil.
 			addPortletURLString = HttpUtil.addParameter(addPortletURLString, "refererPlid", plid);
 			%>
 
-			<aui:nav-item href="<%= addPortletURLString %>" label='<%= LanguageUtil.format(request, "add-x", assetRendererFactory.getTypeName(locale, subtypeSelectionId)) %>' />
+			<aui:nav-item href="<%= addPortletURLString %>" label='<%= LanguageUtil.format(request, "add-x", HtmlUtil.escape(assetRendererFactory.getTypeName(locale, subtypeSelectionId))) %>' />
```

Comment:
portal-web/docroot/html/taglib/aui/nav_item/end.jsp

1. Remove the call to `stripHtml` on line 31 and replace the use of `title` on line 39 with `escapeAttribute`
2. Use `escape` on the label on line 55
3. Do a review of all use cases of <aui:nav-item>, remove any escape on the 'label' attribute